documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx

```diff
@@ -126,31 +131,91 @@ This is a quick summary, to understand the full picture, please see detailed [*R
+
+<Callout type="info" emoji="📌">
+> **active_set_selection_probability = config_score \* ( node_performance ^ 20 ) \* stake_saturation**
+</Callout>
+
+#### Config Score Calculation
+
+The nodes selection to the active set has a new parameter - `config_score`. Config score currently looks into three paramteres:
 
-**Config score is introduced:** The nodes selection to the active set has a new parameter - `config_score`. Config score currently looks if the node binary is `nym-node` (not legacy `nym-mixnode` or `nym-gateway`) **AND** if [Terms & Conditions](nodes/nym-node/setup.mdx#terms--conditions) are accepted. Config score has binary values of either 0 or 1, with a following logic:
+1. If the node binary is `nym-node` (not legacy `nym-mixnode` or `nym-gateway`)
+2. If [Terms & Conditions](nodes/nym-node/setup.mdx#terms--conditions) are accepted.
+3. Version of `nym-node` binary
+
+**The `config_score` parameter calculation formula:**
+
+<Callout type="info" emoji="📌">
+> **config_score = is_tc_accepted \* is_nym-node_binary \* ( 0.995 ^ ( ( X * versions_behind) ^ 1.65 ) )**
+</Callout>
 
-| **Run `nym-node` binary** | **T&C's accepted** | **`config_score`** |
+First two points have binary values of either 0 or 1, with a following logic:
+
+| **Run `nym-node` binary** | **T&C's accepted** | **Value**          |
 | :--                       | :--                | ---:               |
+| True                      | True               | 1                  |
 | True                      | False              | 0                  |
 | False                     | True               | 0                  |
 | False                     | False              | 0                  |
-| True                      | True               | 1                  |
 
+Only if both conditions above are `True` the node can have any chance to be selected, as otherwise the probability will always be 0.
+
+**The `version_behind` parameter in `config_score` calculation**
 
-The entire active set selection probablity:
+From release `2024.14-crunch` (`nym-node v1.2.0`), the `config_score` parameter takes into account also nodes version. Current version is the
+one marked as `Latest` in our repository. From that one we count the parameter `version_behind`, where every version back the number of `versions_behind` increases by 1 in this formula:
 
 <Callout type="info" emoji="📌">
-> **active_set_selection_probability = config_score \* stake_saturation \* node_performance ^ 20**
+> **0.995 ^ ( ( X * versions_behind ) ^ 1.65 )**
+>
+> where: <br />
+> **X = 1;       for patches** <br />
+> **X = 10;      for minor versions** <br />
+> **X = 100;     for major versions**
 </Callout>
 
-For a comparison we made an example with 5 nodes, where first number is node performance and second stake saturation (assuming all of them `config_score` = 1 and not 0):
+> The exact parameters are live accessible on `/v1/status/config-score-details`
```

Comment:
Could make this a link to https://validator.nymtech.net/api/swagger/index.html#/Status/config_score_details